src/telliot_feeds/feeds/__init__.py

```diff
@@ -51,11 +48,9 @@
 }
 
 CATALOG_FEEDS = {
-    "eth-usd-legacy": eth_usd_legacy_feed,
-    "btc-usd-legacy": btc_usd_legacy_feed,
-    "ampl-legacy": ampl_usd_vwap_feed,
-    "uspce-legacy": uspce_feed,
-    "eth-jpy-legacy": eth_jpy_median_feed,
+    "ampl-usd-vwap": ampl_usd_vwap_feed,
```

Comment:
```LEGACY_DATAFEEDS``` is using the same feed, does it need to be changed.  Also should the name change to ```ampl-usd-custom``` to be consistent?

Reply:
I changed the name to be ampleforth-custom, and the other to ampleforth-uspce. I left the feed itself as ampl_usd_vwap, since thhat's what it is and is more specific.